frontend/src/metabase/components/CodeBlock/types.ts

```diff
@@ -0,0 +1,12 @@
+export type CodeLanguage =
+  | "clojure"
+  | "erb"
+  | "html"
+  | "jade"
```

Comment:
since jade and pug are the same thing, I think we can remove the reference to `jade` and replace it with `pug` e.g.
https://github.com/metabase/metabase/blob/cc016e3dd4ad3c0a2532d268ca90da787bb02ad9/frontend/src/metabase/public/lib/code.ts#L22

and maybe other places as well.